### DISCUSSION TITLE
Connection with client certificate

### DISCUSSION THREAD

**Discussion by JunjiUmeno** [kotlinization/kotlin-mqtt-client#1] [Ideas]
TLS not supported yet? Hope it will support connection by client certificate, so clients can connect to AWS IoT or simliar products.

**Comment by MilanJovic92** [comment]
Hi JunjiUmeno,
Thank you for the interest in this library, your suggestion is valid and I would like to add it as a feature.
However I do not have enough time to work on this, so if you would like feel free to create PR and I would be happy to merge it.
